otherlibs/dune-rpc/dune_rpc.ml

```diff
@@ -50,5 +50,11 @@ module V1 = struct
       -> Initialize.t
       -> f:(t -> 'a fiber)
       -> 'a fiber
+
+    val connect_persistent :
+         chan
+      -> on_connect:(unit -> ('a * Initialize.t * Handler.t option) fiber)
+      -> on_connected:('a -> t -> unit fiber)
```

Comment:
These names aren't super descriptive, and it's not immediately obvious what the difference is from the types. Maybe some doc would be good

Reply:
I don't like the names either. Do you have some alternative names in mind?